src/shared/components/msktable/MSKTable.tsx

```diff
@@ -436,27 +437,37 @@ export default class MSKTable<T> extends React.Component<MSKTableProps<T>, {}> {
         />
     }
 
-    render() {
+    buildTextBetweenTheButtons(): string {
```

Comment:
@adamabeshouse Looks like this function depends on `paginationTextSuffix` which is a property of `MSKTable`. It might be better to make it `@computed` within `MSKTable` instead of `MSKTableStore`.